src/neuroconv/tools/nwb_helpers/_backend_configuration.py

```diff
@@ -17,3 +18,17 @@ def get_default_backend_configuration(
 
     BackendConfigurationClass = BACKEND_CONFIGURATIONS[backend]
     return BackendConfigurationClass.from_nwbfile(nwbfile=nwbfile)
+
+
+def get_existing_backend_configuration(nwbfile: NWBFile) -> Union[HDF5BackendConfiguration, ZarrBackendConfiguration]:
+    """Fill an existing backend configuration to serve as a starting point for further customization."""
+
+    read_io = nwbfile.read_io
+    if isinstance(read_io, NWBHDF5IO):
+        backend = "hdf5"
+    elif isinstance(read_io, NWBZarrIO):
+        backend = "zarr"
```

Comment:
Very minor, but you could use this dictionary, which would allow this method to automatically support any new backends added to that dictionary.

https://github.com/catalystneuro/neuroconv/blob/40d786af408e34efaea66edb387fde3d4b8cf32f/src/neuroconv/tools/nwb_helpers/_metadata_and_file_helpers.py#L22

In general, it's better to define this mapping in one single spot. (though you would need to reverse the dictionary first (`reversed_dict = {value: key for key, value in original_dict.items()}`)